runtime/compiler/runtime/MethodMetaData.c

```diff
@@ -313,43 +313,42 @@ static void printMapTable(TR_StackMapTable * stackMapTable, U_8 * addressOfFirst
 static void fastwalkDebug(J9TR_MethodMetaData * methodMetaData, UDATA offsetPC, TR_StackMapTable * stackMapTable, TR_MapTableEntry * mapTableEntry)
    {
    if (methodMetaData && (methodMetaData->flags & JIT_METADATA_IS_STUB))
-      {
       printf("Stub Metadata 0x%p, nothing to dump", methodMetaData);
```

Comment:
I realize it was this way before, but is the `\n` missing in the format string?

Reply:
I don't disagree, but I was making only the compiler error fix here (locals declared after code in C file).

Reply:
Fixed (and another one).